### ISSUE TITLE
s-strings in `table`

### ISSUE THREAD

**Issue by max-sixty** [PRQL/prql#1394]
Currently we allow for very flexible use of s-strings, thanks to #1337 & others:

https://prql-lang.org/book/language-features/s-strings.html#prql-3

But they're not currently permitted in `table`: https://prql-lang.org/book/queries/tables.html#roadmap

```elm
table a = s"""
  SELECT *
  FROM employees
"""

from a
```

My guess is that we can remove this restriction, possible a candidate for a First Issue?

**Comment by BlurrechDev** [comment]
This actually already works - but you have to bracket it explicitly:

```elm
table y = (s"""
  SELECT *
  FROM employees
""")

from y

```

Nonetheless, it would be nice if these could be unambiguously resolved without the brackets.

In my testing, I found it similarly annoying whenever I forgot to bracket:
```elm
from employees
filter employee_id | in (2..6)
```
but I have no idea if that's feasible or a non-starter.

**Comment by max-sixty** [comment]
Thanks for finding that @BlurrechDev !

I looked into the parentheses constraint. The issue is that without parentheses, we don't know where the table stops — if we had

```
table y = from foo
select bar

from employees
```

...then it's unclear whether the `select` is part of the `table`, or there's some other error.

With an s-string:

```
table y = s"""
  SELECT *
  FROM employees
"""

from y
```

...we'd need a special case for parsing just the s-string. Which is not completely impossible, but we're keen to avoid special parsing at the moment, at least until the patterns that end up getting used are better formed.

So I added some docs at https://github.com/PRQL/prql/pull/1413
